src/components/vm/nics/vmNicsCard.jsx

```diff
@@ -281,53 +356,34 @@ export class VmNetworkTab extends React.Component {
                         return _("Unknown");
                     } else {
                         return (
-                            <>
-                                {ips.inet && <div id={`${id}-network-${networkId}-ipv4-address`}>
-                                    {'inet ' + ips.inet}
-                                </div>}
-                                {ips.inet6 && <div id={`${id}-network-${networkId}-ipv6-address`}>
-                                    {'inet6 ' + ips.inet6}
-                                </div>}
-                            </>
+                            <DescriptionList isHorizontal isFluid>
+                                {ips.inet && <DescriptionListGroup>
+                                    <DescriptionListTerm>
+                                        {_("inet")}
+                                    </DescriptionListTerm>
+                                    <DescriptionListDescription id={`${id}-network-${networkId}-ipv4-address`}>
+                                        {ips.inet}
+                                    </DescriptionListDescription>
+                                </DescriptionListGroup>}
+                                {ips.inet6 && <DescriptionListGroup>
+                                    <DescriptionListTerm>
+                                        {_("inet6")}
+                                    </DescriptionListTerm>
+                                    <DescriptionListDescription id={`${id}-network-${networkId}-ipv6-address`}>
+                                        {ips.inet6}
+                                    </DescriptionListDescription>
+                                </DescriptionListGroup>}
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1077-20230626-194339-36f91e04-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)